wdfn-server/waterdata/tests/test_camera.py

```diff
@@ -0,0 +1,94 @@
+from unittest import TestCase
+import sys
+sys.path.append("..")
+import camera
+# from .. import camera
```

Comment:
I would prefer that this and other snippets of commented out code (e.g the `print`s) be removed before merging.

Reply:
They are only in a unit test, not in main code.

Reply:
Unless there is a compelling reason to leave in commented out code, it should be removed, IMO